metrics/storage/blogbench_dockerfile/Dockerfile.in

```diff
@@ -15,13 +15,17 @@ ENV BLOGBENCH_URL "https://download.pureftpd.org/pub/blogbench"
 ENV BLOGBENCH_VERSION 1.1
 
 RUN apt-get update && \
-	apt-get install -y build-essential curl && \
+	apt-get install -y --no-install-recommends build-essential curl && \
 	apt-get remove -y unattended-upgrades && \
-	curl -OkL ${BLOGBENCH_URL}/blogbench-${BLOGBENCH_VERSION}.tar.gz && \
-	tar xzf blogbench-${BLOGBENCH_VERSION}.tar.gz && \
-	cd blogbench-${BLOGBENCH_VERSION} && \
-	export arch=$(uname -m) && \
-	./configure --build=${arch} && \	
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/ && \
+	curl -OkL "${BLOGBENCH_URL}/blogbench-${BLOGBENCH_VERSION}.tar.gz" && \
+	tar xzf "blogbench-${BLOGBENCH_VERSION}.tar.gz -C /"
```

Comment:
The quotes are a bit odd here - it would make more sense to me if the end quote finished at `.tar.gz"`.

Reply:
True. I am going to fix it.